src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -111,14 +111,13 @@ Template:
 ```Text
 sub-<participant_label>/[ses-<session_label>/]
     anat/
-        sub-<participant_label>[_ses-<session_label>][_acq-<label>][_ce-<label>][_rec-<label>][_run-<index>]_<modality_label>.nii[.gz]
+        sub-<participant_label>[_ses-<session_label>][_acq-<label>][_ce-<label>][_rec-<label>][_run-<index>]_<contrast_label>.nii[.gz]
         sub-<participant_label>[_ses-<session_label>][_acq-<label>][_ce-<label>][_rec-<label>][_run-<index>][_mod-<label>]_defacemask.nii[.gz]
 ```
 
-Anatomical (structural) data acquired for that participant. Currently supported
-modalities include:
+Currently supported image contrasts include:
 
-| Name               | modality_label | Description                                                                                                                                                                                                                             |
+| Name               | contrast_label | Description                                                                                                                                                                                                                             |
```

Comment:
Changes to the anatomical section seem tangential and I would leave them to another PR to keep the discussion focused.

Reply:
Depends how widely you propose it to apply - if for the whole spec then in definitions.

Please move these changes to a separate pull request - I expect this will raise a longer discussion which should not slow down merging changes related strictly to CBV.

Reply:
ok